pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[tool.poetry]
+name = "astrapy"
+version = "0.6.3"
```

Comment:
At the moment, this is duplicated with `__version__` but there should be ways to only use this info.
For instance we can now get the installed package version with `importlib.metadata.version('astrapy')`

Reply:
@cbornet This is great. could you look at the commit I just made? I think this will handle the issue. Let me know!